Automattic/jetpack
# Assistant: Change illustration in Site Type step in Assistant

This PR aims to change/update the illustration used in the Site Type step in the Jetpack Assistant in accordance to the updated design in p6TEKc-5bW-p2.

### Changes proposed in this Pull Request:

* Change the Site Type illustration in the Jetpack Assistant.
* Prettify the image path for the Site Type illustration in the Jetpack Assistant.

**Change the Site Type illustration in the Jetpack Assistant**

We replace the existing illustration used in the Site Type step, in the Assistant, that maintains the same dimensions and general position of elements so it doesn't require any styling updates.

**Prettify the image path for the Site Type illustration in the Jetpack Assistant**

The image path would contain a `//` due to two concatenated strings both containing prefixed/suffixed slashes.
`wp-content/plugins/jetpack/images//recommendations/site-type-illustration.jpg`
=>
`wp-content/plugins/jetpack/images/recommendations/site-type-illustration.jpg`

### Jetpack product discussion



* p6TEKc-5bW-p2
* pbNhbs-Yp-p2

### Does this pull request change what data or activity we track or use?



No.

### Testing instructions:






* Go to `/wp-admin/admin.php?page=jetpack#/recommendations/site-type`
* Verify that the illustration used on the linked page matched the "After" screenshot form the section below.

### Screenshots

**Before**

![Screenshot 2021-08-12 at 14 11 31](https://user-images.githubusercontent.com/3846700/129195845-b765f7ee-2679-4988-8e02-0b40a587508a.png)

**After**

![Screenshot 2021-08-12 at 14 11 01](https://user-images.githubusercontent.com/3846700/129195865-917b4abf-9a92-43ba-bdd2-9f9d3acefd32.png)
